pyiron_atomistics/atomistics/structure/analyse.py

```diff
@@ -206,3 +207,41 @@ def pyscal_voronoi_volume(self):
         """    Calculate the Voronoi volume of atoms        """
         return analyse_voronoi_volume(atoms=self._structure)
 
+    def get_voronoi_vertices(self, epsilon=2.6544356738490314e-4, distance_threshold=0, width_buffer=10):
```

Comment:
If the purpose is to find interstitial sites, I believe we can just change the name to reflect that and mention in the docs that it uses a Voronoi teselation. 

Also where's that specific epsilon coming from?

Reply:
In bcc it doesn't find the octahedral interstitial sites, which is deadly for Fe-C.

The epsilon value comes from my notebook. I just wanted to have a fixed small random number.

Reply:
Could we just use `2.5e-4`? This way it looks like it might be important to get the digits right.